tests/UnitTests/UnitTests.csproj

```diff
@@ -1,34 +1,39 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
-		<PackageReference Include="Microsoft.Orleans.Server" Version="9.1.2" />
-		<PackageReference Include="RavenDB.Embedded" Version="7.0.0" />
-		<PackageReference Include="Microsoft.Orleans.CodeGenerator" Version="9.1.2">
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.14.1" />
+		<PackageReference Include="Microsoft.Orleans.Server" Version="9.2.0-preview3" />
+		<PackageReference Include="Microsoft.Orleans.TestingHost" Version="9.2.0-preview3" />
+		<PackageReference Include="MSTest.TestAdapter" Version="3.9.3" />
+		<PackageReference Include="MSTest.TestFramework" Version="3.9.3" />
+		<PackageReference Include="RavenDB.Embedded" Version="7.1.0-rc-71004" />
+		<PackageReference Include="Microsoft.Orleans.CodeGenerator" Version="9.2.0-preview3">
```

Comment:
same here, do we actually need all of this?
what is `MSTest` used for?

Reply:
done